pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
@ayoayco upps, you should update `pnpm` via `corepack`, this version is wrong, can you run `pnpm --version`  in your local?

I'll try to fix it tmr.

Reply:
I don't check it since I always run `pnpm install --frozen-lockfile`, we should exclude the lockfile in this PR.

Reply:
I pushed a revert for this lock file